### PR TITLE
[RUM-998] Replace issue md files with yaml based forms

### PR DESCRIPTION
**Pull request by plousada**
### What does this PR do?

Replaces the current markdown files with more detailed yaml based forms.

Created forms for:
- Bug Report
- Crash Report
- Feature Request
- Setup Issue
- Question

Using forms will create a more pleasant and organized UI versus just using markdown files.

### Motivation

The main objective is to have more detailed issues and cut down on time spent gathering information.

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)
- [ ] If this PR is auto-generated, please make sure also to manually update the code related to the change

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BugReport.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        For a more complex setup consider creating a small app that showcases the problem. 
+        **Note** - Avoid sharing any business logic, credentials or tokens.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: SDK logs
+      description: |
+        Please provide console logs before, during and after the bug occurs.
```
Comment (louiszawadzki) [inline .github/ISSUE_TEMPLATE/BugReport.yml]: The important information for troubleshooting is often in the native logs, rarely in the JS console logs. 
So I'd suggest to mention them explicitely, with a link explaining how to get them. Do you think that's a good idea or too much to ask for?

```suggestion
        Please provide javascript and native logs from before, during and after the bug occurs. Refer to [our troubleshooting page](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#3-review-native-logs) on how to collect native logs.
```

File: .github/ISSUE_TEMPLATE/BugReport.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        - NPM
+        - Yarn
+        - Source
+    validations:
+        required: true
+  - type: input
+    id: rn_version
+    attributes:
+      label: React Native Version
+      description: e.g. 0.73
```
Comment (louiszawadzki) [inline .github/ISSUE_TEMPLATE/BugReport.yml]:
```suggestion
      description: e.g. 0.73.3
```
We already had bugs on patch versions of RN, so I think we should include the patch as well in the example, what do you think?

File: .github/ISSUE_TEMPLATE/CrashReport.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+  - type: input
+    id: rn_version
+    attributes:
+      label: React Native Version
+      description: e.g. 0.73
```
Comment (louiszawadzki) [inline .github/ISSUE_TEMPLATE/CrashReport.yml]:
```suggestion
      description: e.g. 0.73.3
```

File: .github/ISSUE_TEMPLATE/SetupIssue.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        - NPM
+        - Yarn
+        - Source
+    validations:
+        required: true
+  - type: input
+    id: rn_version
+    attributes:
+      label: React Native Version
+      description: e.g. 0.73
```
Comment (louiszawadzki) [inline .github/ISSUE_TEMPLATE/SetupIssue.yml]:
```suggestion
      description: e.g. 0.73.3
```

**Review by louiszawadzki — DISMISSED** [review]
Some minor improvements suggested, looks already good to me!

File: .github/ISSUE_TEMPLATE/BugReport.yml (hunk 1 above)
Comment (plousada) [reply .github/ISSUE_TEMPLATE/BugReport.yml]: No, it's perfectly reasonable and likely something we'd end up doing via additional comments.
Good catch.

**Review by louiszawadzki — APPROVED** [review]

**Pull request merged**